hoodie-spark/src/main/scala/com/uber/hoodie/HoodieSparkSqlWriter.scala

```diff
@@ -117,7 +117,7 @@ private[hoodie] object HoodieSparkSqlWriter {
     }
 
     // Create the dataset if not present (APPEND mode)
-    if (!exists) {
+    if (!fs.exists(new Path(basePath, HoodieTableMetaClient.METAFOLDER_NAME))) {
```

Comment:
should the `exists` variable above change instead? and we keep this line as is?

Reply:
done